src/components/node-list/node-list-row-list.js

```diff
@@ -4,6 +4,19 @@ import { connect } from 'react-redux';
 import NodeListRow, { nodeListRowHeight } from './node-list-row';
 import LazyList from '../lazy-list';
 
+// modify display of labels for modular pipelines to show nested relationship
+const getItemLabel = (item) => {
+  const layer = '・';
+  const whiteSpace = '&nbsp;&nbsp;&nbsp;&nbsp;';
+  if (item.type === 'modularPipeline') {
+    const levels = item.id.match(/\./g) ? item.id.match(/\./g).length : 0;
+    return `${whiteSpace
+      .repeat(levels)
+      .concat(layer.repeat(levels).concat(item.highlightedLabel))}`;
```

Comment:
Are more deeply nested pipelines supposed to have more than one dot? I thought they would just have a single dot, and that the number of spaces should determine the nesting level?

![image](https://user-images.githubusercontent.com/1155816/114609142-60eb6880-9c96-11eb-8ccc-634da776889e.png)

Reply:
great spot, I think I added this on following the comment at some point on the slack thread and we decided not to go for the 2 dots. I'll take out the multiple dot in the formatting

Reply:
I've updated the label formatting and it now looks like this: 
![image](https://user-images.githubusercontent.com/27775658/114738124-9730f280-9d3f-11eb-985a-fb34521d8b68.png)